cognite/neat/_session/_inspect.py

```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+
+from ._state import SessionState
+from .exceptions import intercept_session_exceptions
+
+
+@intercept_session_exceptions
+class InspectAPI:
+    def __init__(self, state: SessionState) -> None:
+        self._state = state
+
+    @property
+    def properties(self) -> pd.DataFrame:
+        """Returns the properties of the current data model."""
+        return self._state.last_verified_rule.properties.to_pandas()
```

Comment:
I've added Neat session error for last_verified rules so you have to check here if `self._state.last_verified_rule` does not raise error prior doing this

Reply:
I think we want to raise an error here if last_verified_rule is not there.